examples/rhscl_maria/example1.py

```diff
@@ -0,0 +1,119 @@
+
+from moduleframework import module_framework
+from moduleframework.helpers.container_helper import ContainerHelper
+from moduleframework import common
+from avocado.utils import process
+from avocado import Test
+import time
+
+WAIT_TIME=10
+
+class OneMachine(module_framework.AvocadoTest):
+    """
+    Basic, the most straightforward example how to use it with collections,
```

Comment:
It is just for containers. I should use module_framework.ContainerAvocadoTest instead of generic one. I'll change it.